persistence/sql/persister_continuity.go

```diff
@@ -41,3 +42,17 @@ func (p *Persister) DeleteContinuitySession(ctx context.Context, id uuid.UUID) e
 	}
 	return nil
 }
+
+func (p *Persister) DeleteExpiredContinuitySessions(ctx context.Context, expiresAt time.Time) error {
+	// #nosec G201
+	err := p.GetConnection(ctx).RawQuery(fmt.Sprintf(
+		"DELETE FROM %s WHERE expires_at <= ?",
+		new(continuity.Container).TableName(ctx),
+	),
+		expiresAt,
+	).Exec()
```

Comment:
```suggestion
	err := p.GetConnection(ctx)
		.Where("expires_at <= ?", expiresAt)
		.Delete(new(continuity.Container)),
```

Reply:
The problem with this query is that it will be extremely slow as it is unbound. Check out this PR for batch-based processing: https://github.com/grantzvolsky/hydra/pull/2/files#diff-6034803b09ef5017e3aa7d3082827dadb4503e6ee3a56853d49edd419a75f864